.github/workflows/ci.yml

```diff
@@ -16,8 +16,8 @@ on: [push, pull_request]
 # NOTE: Testing the full matrix is not practical.
 # Therefore we aim to have each value been set in at lest one job.
 # CXX                                           : {g++, clang++}
-#   [g++] ALPAKA_CI_GCC_VER                     : {7, 8, 9, 10}
-#   [clang++] ALPAKA_CI_CLANG_VER               : {5.0, 6.0, 7, 8, 9, 10, 11}
+#   [g++] ALPAKA_CI_GCC_VER                     : {7, 8, 9, 10, 11}
+#   [clang++] ALPAKA_CI_CLANG_VER               : {5.0, 6.0, 7, 8, 9, 10, 11, 12}
```

Comment:
Why not add clang 13, too?

Reply:
Since I was messing with these descriptions anyway, I just updated the version here to match the jobs we already had. We should add clang-13 in a separate PR. See:  #1476.